lib/bricks-json.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var when = require('when');
+
+
+exports.JSON = function (object, name) {
+	return when(object).then(function (ret) {
+		var object = ret;
```

Comment:
I'd suggest we follow Cereal's approach for now and always keep the top-level object, ex., `"value0"`.